pageserver/src/metrics.rs

```diff
@@ -548,15 +548,66 @@ const STORAGE_IO_TIME_BUCKETS: &[f64] = &[
 /// - seek (modify internal position or file length query)
 /// - fsync ([`std::fs::File::sync_all`])
 /// - metadata ([`std::fs::File::metadata`])
-pub(crate) static STORAGE_IO_TIME: Lazy<HistogramVec> = Lazy::new(|| {
-    register_histogram_vec!(
-        "pageserver_io_operations_seconds",
-        "Time spent in IO operations",
-        &["operation"],
-        STORAGE_IO_TIME_BUCKETS.into()
-    )
-    .expect("failed to define a metric")
-});
+#[derive(
```

Comment:
I think we no longer need this comment listing which is confusing (me at least, in https://github.com/neondatabase/neon/pull/5273/files#r1321287671) -- it was added after some comment about "it'd be nice to have all of the variants" explained, but now we do have them, as proper enum variants.

Reply:
Or it could be moved to be per variant docs but ... These are quite self-explanatory, at except maybe `CloseByReplace`.

Reply:
Well, this is no biggie, we can handle it later.